extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1855,13 +1855,11 @@ export class AmpStoryPage extends AMP.BaseElement {
   initializeImgAltTags_() {
     toArray(this.element.querySelectorAll('amp-img')).forEach((ampImgNode) => {
       if (!ampImgNode.getAttribute('alt')) {
-        ampImgNode.setAttribute('alt', ' ');
-        // If the child img element is in the dom, propogate the attribute to it.
+        ampImgNode.setAttribute('alt', '');
+        // If the child img element is in the dom, put empty alt on it.
         const childImgNode = ampImgNode.querySelector('img');
         childImgNode &&
-          ampImgNode
-            .getImpl()
-            .then((ampImg) => ampImg.propagateAttributes('alt', childImgNode));
+          ampImgNode.getImpl().then(() => childImgNode.setAttribute('alt', ''));
```

Comment:
Just to understand, why don't we want to propagate the attr? Because the underlying image already will read it?

Reply:
Reverted that. Current implementation is working. Thank you for asking this.